eli5/catboost.py

```diff
@@ -0,0 +1,84 @@
+
+from __future__ import absolute_import,division
+
+import numpy as np
+import catboost
+
+from eli5.explain import explain_weights
+from eli5._feature_importances import get_feature_importance_explanation
+
+DESCRIPTION_CATBOOST = """
+CatBoost feature importances; values are numbers 0 <= x <= 1;
+all values sum to 1.
+"""
+
+
+@explain_weights.register(catboost.CatBoost)
+@explain_weights.register(catboost.CatBoostClassifier)
+@explain_weights.register(catboost.CatBoostRegressor)
+def explain_weights_catboost(catb,
+                            vec=None,
+                            top=20,
+                            importance_type='PredictionValuesChange',
+                            prettified=True,
+                            feature_names=None,
+                            Pool=None
```

Comment:
I'd suggest changing the name to lowercase ``pool``, and documenting this parameter in the docstring.